src/translations/en.json

```diff
@@ -5379,9 +5382,12 @@
         "title": "Network settings",
         "connected_to": "Connected to {ssid}",
         "scan_ap": "Scan for access points",
+        "signal_strength": "Signal strength",
```

Comment:
Should reference the other key here

Reply:
It can't here because those are not loaded at runtime for those supervisor panels. They use different localizer objects.

Reply:
The `[%key:...%]` references are resolved when the translations are downloaded from Lokalise before the build.

Reply:
True, brain fart on my end. We need the different sections in the keys, that is where the localizer object for Supervisor comes into play.

But I am wondering now why all the existing duplicate keys in that translation section do not share translations. Was there perhaps a reason here?

Reply:
Probably a quick copy/paste, plus policy to avoid duplicates is not strictly enforced.  Would be a good thing to add to CI IMO.

Reply:
Probably a quick copy/paste indeed. If you can fix the key reference here for signal strength, we can merge this. The other keys can happen in another PR.